flang/lib/Optimizer/Builder/Runtime/Main.cpp

```diff
@@ -61,6 +65,15 @@ void fir::runtime::genMain(
   args.push_back(env);
 
   builder.create<fir::CallOp>(loc, startFn, args);
+
+#ifdef FLANG_CUDA_SUPPORT
+  if (initCuda) {
+    auto initFn = builder.createFunction(
```

Comment:
What should happen in the following cases?
 * Flang is compiled with `FLANG_CUDA_SUPPORT`, compiles a CUDA program, which is then executed in an environment without CUFRuntime. 
 *  Flang is compiled without `FLANG_CUDA_SUPPORT`, compiles a CUDA program which is then executed in an environment without CUFRuntime.
 *  Flang is compiled without `FLANG_CUDA_SUPPORT`, compiles a CUDA program which is then executed in an environment that supports CUDA.